src/pages/patientView/clinicalInformation/lib/clinicalAttributesUtil.js

```diff
@@ -207,21 +207,56 @@ function getSpanElementsFromCleanData(
     clinicalAttributesCleanDerived,
     cancerStudyId
 ) {
-    let spans = [];
-    return Object.keys(clinicalAttributesCleanDerived).map(key => {
-        let value = clinicalAttributesCleanDerived[key];
-        return (
-            <span
-                is
-                class="clinical-attribute"
-                attr-id={key}
-                attr-value={value}
-                study={cancerStudyId}
-            >
-                {value}
-            </span>
-        );
-    });
+    // Order patient clinical attributes
+    const getOrder = key => {
+        switch (key) {
```

Comment:
instead of a switch statement, i would make a map: {  'SEX':1, ....   }, then check in this function for undefined and return the default if it's undefined.

Reply:
Ok, I'm currently refactoring the code to use an object instead of a switch statement. 
(I'm currently having trouble since I can't subscript the object with the function argument, but I will keep you updated!)

Reply:
I've just pushed another commit: "Edited patient view logic based on Aaron's suggestion."